locust/main.py

```diff
@@ -462,7 +474,21 @@ def sig_term_handler():
         logger.info("Got SIGTERM signal")
         shutdown(0)
     gevent.signal(signal.SIGTERM, sig_term_handler)
-    
+
+    # interval updater
+    def do_interval_update(update_interval):
+        update_times = 1
+        while True:
+            try:
+                events.locust_update.fire(times=update_times)
+            except Exception, e:
```

Comment:
change `except Exception, e:`  to `except Exception as e:`
only the newer exception syntax is valid in Python3